contrib/examples/create-cluster.sh

```diff
@@ -20,12 +20,14 @@ oc get secret aws-credentials -n cluster-operator -o yaml | sed -E '/(namespace:
 oc get secret ssh-private-key -n cluster-operator -o yaml | sed -E '/(namespace:|annotations|last-applied-configuration:|selfLink|uid:|resourceVersion:)/d' | oc apply -f -
 oc get secret ssl-cert -n cluster-operator -o yaml | sed -E '/(namespace:|annotations|last-applied-configuration:|selfLink|uid:|resourceVersion:)/d' | oc apply -f -
 
+default_name="$(whoami)-cluster"
+name="${1:-$default_name}"
 
-if [ -e contrib/examples/$(whoami)-cluster.yaml ]
+if [ -e contrib/examples/${name}.yaml ]
```

Comment:
I would rather that the user specified the file path rather than inferring the yaml extension and the location. I don't like that the name of the cluster created is inextricably tied to the name of the yaml file. I would rather continue to use `${whoami}-cluster` as the value passed to CLUSTER_NAME, where the user can choose to ignore that parameter in the custom template that they are using.